yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -203,16 +203,18 @@ func checkFileFormat() {
 
 func checkDataDirFlag() {
 	if dataDir == "" {
-		fmt.Fprintln(os.Stderr, `ERROR: required flag "data-dir" not set`)
-		os.Exit(1)
+		utils.ErrExit(`ERROR: required flag "data-dir" not set`)
 	}
 	if !utils.FileOrFolderExists(dataDir) {
-		fmt.Fprintf(os.Stderr, "Directory: %s doesn't exists!!\n", dataDir)
-		os.Exit(1)
-	} else if dataDir == "." {
+		utils.ErrExit("Directory: %s doesn't exists!!", dataDir)
+	}
+	dataDir = filepath.Clean(dataDir)
+	if filepath.Dir(dataDir) == exportDir || dataDir == filepath.Clean(exportDir) {
+		utils.ErrExit("ERROR: Either --export-dir or --data-dir flag is invalid, both needs to be at different location/directory")
```

Comment:
if the data directory is anywhere within the export directory, the error will arise, please look at the details of the issue I had raised (#319) for an example of this. It does not necessarily only mention the case where data-dir and export-dir are the same

Reply:
I think `filePath.Dir()` returns get the parent directory of `dataDir` and this condition checks it if parent directory of `dataDir` is `exportDir` or not, I think this will handle that case where `dataDir` is within `exportDir` . @sanyamsinghal let me know if this understanding of mine is wrong?

Reply:
yes @priyanshi-yb 
@rahulb-yb I am handling both the cases if they are the same or dataDir is within exportDir. So i think this should the case mentioned in your ticket. let me know if i am missing something.

Reply:
Can't we just check if export dir is prefix of data dir?
```
if strings.HasPrefix(filepath.Abs(dataDir), filepath.Abs(exportDir)) {
  // ERROR: data dir must be outside of export dir
}
```

Reply:
yes @amit-yb we can do that. But I think this code is more readable and meaningful

Reply:
From the code it looks like you're only checking two cases (ignoring path cleaning):
- export dir == data dir
- data dir is an immediate child of export dir

As rahul is suggesting, the issue can happen whenever data dir is anywhere in the export-dir tree.

Reply:
I understand the prefix logic but do we really need to create an error for all those cases - what if data dir is in `temp` or `reports` directory inside the `export-dir`, technically that's not a problem.

Reply:
Okay. Let's go with your approach.